### ISSUE TITLE
Accessibility issue: Social media links are missing discernible text for screen readers.

### ISSUE THREAD

**Issue by mi5t4n** [rtCamp/godam#425]
## Description
Social links within the admin pages lack descriptive text for screen readers.

![Image](https://github.com/user-attachments/assets/f0a19731-cd82-44dd-9089-50530ccf25f6)

**Comment by kishan-gondaliya-7270** [comment]
@mi5t4n This doesnt fix accessibility issue at all just adds aria-label which is one of the measure for a11y. However there are use of aria-label where it is not permitted according to axe scan. 

<img width="1398" height="551" alt="Image" src="https://github.com/user-attachments/assets/d216b444-3696-474c-acec-8400b5dbb835" />
